easy-ui-react/package.json

```diff
@@ -33,6 +33,7 @@
     "@vitejs/plugin-react": "^3.1.0",
     "jsdom": "^21.1.0",
     "react": "^18.2.0",
+    "react-aria": "^3.23.1",
```

Comment:
is this okay as a dev dep or should it be a standard dep?

Reply:
it should actually be a standard dep since it'll be "included" in the lib. in contrast, react is a peer dependency that's excluded from the build, for instance. we'll also want to make sure Vite is not including all of react-aria during build. you can test this by running `npm run build:react` and looking at the file sizes that are output. if it's huge, it's likely including the entire library instead of tree shaking it

Reply:
added as regular dep, i compared the size of `dist` when using `react-aria` vs `@react-aria/button` and they appear to be the same, i presume that means the library was correctly tree shaken